bom/pom.xml

```diff
@@ -38,6 +38,8 @@
     <influxdb.version>1.2.4-alpine</influxdb.version>
     <jackson.version>2.9.5</jackson.version>
     <java-base-image.name>openjdk:8u171-jre-slim</java-base-image.name>
+    <jaxb.api.version>2.3.0</jaxb.api.version>
```

Comment:
version 2.2.12 also does the trick and we could simply piggy back https://dev.eclipse.org/ipzilla/show_bug.cgi?id=13115

Reply:
Well I wanted to be a good citizen and create a CQ for the most recent version for other to PB on. :wink: